app/src/main/java/io/gnosis/safe/ui/safe/add/AddSafeNameViewModel.kt

```diff
@@ -12,7 +11,8 @@ import javax.inject.Inject
 class AddSafeNameViewModel
 @Inject constructor(
     repositories: Repositories,
-    appDispatchers: AppDispatchers
+    appDispatchers: AppDispatchers,
+    private val tracker: Tracker
```

Comment:
Maybe it would be good to have this as a singleton already to not have to inject it everywhere.